python/cudf/cudf/_lib/copying.pyx

```diff
@@ -204,7 +204,7 @@ def _scatter_scalar(scalars, Column scatter_map,
     cdef Scalar slr
     for val, col in zip(scalars, target_table._columns):
         slr = as_scalar(val, col.dtype)
-        source_scalars.push_back(move(slr.c_value))
+        source_scalars.push_back(move(slr.get_c_ptr()))
```

Comment:
The name of this is a bit confusing since this returns a unique ptr as opposed to a raw ptr